docs/configuration.md

```diff
@@ -312,7 +313,7 @@ In some cases the host value is not present in an event (or an IP address is in
 who require a true hostname be attached to each event, SC4S provides an optional facility to perform a reverse IP to
 name lookup. If the variable `SC4S_USE_REVERSE_DNS` is set to "yes", SC4S
 will first check `host.csv` and replace the value of `host` with the value specified that matches the incoming IP address.
-If a value is not found in `host.csv` then a reverse DNS lookup will be attempted against the configured nameserver.
+If a value is not found in `host.csv` then a reverse DNS lookup will be attempted against the configured nameserver. In this case, SC4S by default extracts only the hostname from FQDN (`example.domain.com` -> `example`). If `SC4S_REVERSE_DNS_KEEP_FQDN` variable is set to "yes", full domain name will be assigned to the host field. 
 The IP address will only be used as the host value as a last resort.
 
 * NOTE:  Use of this variable can have a significant impact on performance if the reverse DNS facility (typically a caching
```

Comment:
This variable can have a significant impact on performance if the reverse DNS facility (typically a caching nameserver) is not performant. If you notice events being indexed far later than their actual timestamp in the event (latency between `_indextime` and `_time`), you should check this variable first.